src/OctoshiftCLI.Tests/HttpDownloadServiceTests.cs

```diff
@@ -79,15 +73,19 @@ public async Task Raises_Exception_When_File_Cannot_Be_Downloaded()
 
             using var httpClient = new HttpClient(mockHttpHandler.Object);
 
-            // Act
-            var httpDownloadService = new HttpDownloadService(_mockOctoLogger.Object, httpClient)
+            var path = System.IO.Path.GetTempPath() + "integration_test";
+
+            if (File.Exists(path))
             {
-                WriteToFile = (_, contents) =>
-                {
-                    fileContents = contents;
-                    return Task.CompletedTask;
-                }
-            };
+                File.Delete(path);
+            }
+
+            using var fs = File.Create(path);
+
+            _mockFileSystemProvider.Setup(x => x.Open(filePath, System.IO.FileMode.Open)).Returns(fs);
```

Comment:
I guess you missed to update this, just like the `Download_File` test all we need to do is to return any `FileStream`:

```c#
            _mockFileSystemProvider.Setup(x => x.Open(filePath, System.IO.FileMode.Open)).Returns(It.IsAny<FileStream>());
```

Reply:
Addressed in 7d436e04071bf868990e1d32d6276668c1db1a35 :tada: